monitoring/rid_qualifier/aircraft_state_replayer.py

```diff
@@ -0,0 +1,133 @@
+import requests
+from requests.sessions import get_auth_from_url
+from monitoring.monitorlib.auth import make_auth_adapter
+from monitoring.monitorlib.infrastructure import DSSTestSession
+import asyncio
+from monitoring.monitorlib import rid
+import json, os
+import uuid
+from pathlib import Path
+from typing import  Any
+from utils import OperatorLocation, RIDFlightDetails, TestFlightDetails, TestFlight
+from urllib.parse import urlparse
+from monitoring.monitorlib.rid import SCOPE_READ, SCOPE_WRITE
```

Comment:
nit: This separate import is unnecessary; we already import `rid` above, and then we refer to `rid.SCOPE_READ` and `rid.SCOPE_WRITE`.

Reply:
Got it! I have now removed this!